backend/src/app/participant/get-participant-index.usecase.ts

```diff
@@ -0,0 +1,16 @@
+import { IParticipantQS } from './query-service-interface/participant.qs'
+
+export class GetParticipantIndexUseCase {
+  private readonly participantDataQS
+  constructor(participantDataQS: IParticipantQS) {
+    this.participantDataQS = participantDataQS
+  }
```

Comment:
```suggestion
  constructor(private readonly participantDataQS: IParticipantQS) {}
```

[Constructor Shorthand](https://typescriptbook.jp/reference/object-oriented/class/constructor-shorthand) の綺麗かも。

Reply:
とおもったら、他はそうしてた。笑
失礼いたしました。

Reply:
統一して修正します！